src/app/pages/administration/jobs/user-triggers-view.jsx

```diff
@@ -78,7 +78,7 @@ function ChplUserTriggersView(props) {
           { (triggers.length === 0)
             && (
               <Typography className={classes.noResultsContainer}>
-                No results found
+                No results found. To get started, select the type of job then click on the calender icon to set-up a schedule job.
```

Comment:
Has this text gone past the PO? This looks like a functional change, though admittedly a small one

Reply:
No this hasnt, need a PO to approve